articles/azure-functions/security-concepts.md

```diff
@@ -221,11 +221,11 @@ FTP isn't recommended for deploying your function code. FTP deployments are manu
 
 When you're not planning on using FTP, you should disable it in the portal. If you do choose to use FTP, you should [enforce FTPS](../app-service/deploy-ftp.md#enforce-ftps).
 
-### Secure the scm endpoint
+### Secure the SCM endpoint
```

Comment:
From what I can ascertain, `scm` isn't a declared TLA, so maybe we can style as code instead?
```suggestion
### Secure the `scm` endpoint
```

Reply:
Hi, sorry for not replying so long. 
I checked with other articles, for example, on App Service, it usually says "SCM" without code formatting. Should we also fix those instances for consistency?